tests/loggers/test_all.py

```diff
@@ -56,7 +56,8 @@ def _instantiate_logger(logger_class, save_idr, **override_kwargs):
     return logger
 
 
-def test_loggers_fit_test_all(tmpdir, monkeypatch):
+@mock.patch("torch.save")
+def test_loggers_fit_test_all(torch_save_mock, tmpdir, monkeypatch):
```

Comment:
I added the `torch.save` mock. This prevents saving checkpoints and does not pickle.
In this particular set of tests we can't pickle the logger class, because it's getting defined in the helper function below.
Checkpointing is not essential for this test here, we have others that do logging + checkpointing.